### PR TITLE
Right Click Menu: Clear multiple element styles

### LINKED ISSUE #9885: Right Click Menus: Multiple Elements Clear Styles
## Context 

Separated from user feature request #4616 to make sure initial new menus for when multiple elements are selected has isolated test time.

## Feature Description 

When multiple elements are selected and a user right clicks they should get the option to 'clear styles' for all elements selected. 

## Acceptance Criteria 
- New menu option for multiple element right click menu to clear styles that will clear styles for all those elements (same logic as the clear styles for single elements that we have right now) 
- Corresponding tests

### PR DESCRIPTION
**Pull request by samwhale**
## Context

Separated from user feature request #4616 to make sure initial new menus for when multiple elements are selected has isolated test time.

Branch from #9832. Once that is merged this will be smaller.

## Summary

Adds functionality to clear styles for multiple elements at the same time using the context menu.

## Relevant Technical Choices

Elements that couldn't be reset (text) still may not be reset even if selected.

## To-do

n/a

## User-facing changes

clear-styles

## Testing Instructions

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. Add multiple elements to the canvas of different types. Change their styles.
2. Select them all using the `Shift` key
3. Right click on any of the selected elements or the frame
4. Click `Clear Styles`. Should see any image and shape elements get reset to defaults. All text elements should not be changed.
5. Click `Undo` in the snackbar that shows up
6. Should see element styles all revert to what they originally were.
7. `Undo` should work with the keyboard shorcut `mod + z`

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #9885

### CODE REVIEW

**Comment by samwhale** [comment]
Listed this as blocked because it's sharing code from #9832 which is in QA. Once that is merged this will be much smaller.

**Review by BrittanyIRL — APPROVED** [review]
lgtm! thank you for the updated notes, this feature is so well documented 😻 

re the karma test failures, miina pointed those out today and disabled some of them so hopefully rebasing will give you a pass. they don't seem related to this pr at all.

File: packages/story-editor/src/app/rightClickMenu/provider.js [hunk 1]
```diff
@@ -599,20 +628,24 @@ function RightClickMenuProvider({ children }) {
 
           trackEvent('context_menu_action', {
             name: 'undo_clear_styles',
-            element: selectedElementType,
-            isBackground: selectedElement?.isBackground,
+            elements: [selectedElements.map((element) => element.type)],
```
Comment (maxyinger) [inline packages/story-editor/src/app/rightClickMenu/provider.js]: should this be a nested array or nah?
```suggestion
            elements: selectedElements.map((element) => element.type),
```

File: packages/story-editor/src/app/rightClickMenu/provider.js [hunk 2]
```diff
@@ -599,20 +628,24 @@ function RightClickMenuProvider({ children }) {
...
           });
         },
         actionHelpText: UNDO_HELP_TEXT,
       });
 
       trackEvent('context_menu_action', {
         name: 'clear_styles',
-        element: selectedElementType,
-        isBackground: selectedElement?.isBackground,
+        elements: [selectedElements.map((element) => element.type)],
```
Comment (maxyinger) [inline packages/story-editor/src/app/rightClickMenu/provider.js]: same as above. should this be a nested array?

**Review by maxyinger — APPROVED** [review]

File: packages/story-editor/src/app/rightClickMenu/provider.js (hunk 1 above)
Comment (samwhale) [reply packages/story-editor/src/app/rightClickMenu/provider.js]: good catch - this should not be a nested array

**Review by BrookeGraham — APPROVED** [review]
lgtm

**Comment by BrittanyIRL** [comment]
This code was pulled and tested during peer review by 3 people, merging to avoid stale queue.

**Pull request merged**